### ISSUE TITLE
Error during evaluation - used PASCAL VOC dataset

### ISSUE THREAD

**Issue by Mahmood-Hussain** [oandrienko/fast-semantic-segmentation#35]
Thanks for your work.
I tried this source code on the PASCAL VOC dataset (converted it to .tfrecord format using this code [https://github.com/tensorflow/models/tree/master/research/deeplab](https://github.com/tensorflow/models/tree/master/research/deeplab))

The training was successful and the loss at the end of 40000 steps was 1.32 around when I tried to evaluate it throws me errors kindly see the screenshot

I searched internet and looks there may be an issue with the version of tensorflow-gpu I am using help from you would be appreciated or if you can tell me which version of TensorFlow to use that is also fine. Tried TensorFlow 1.8 as well but that also did  not work

ENVIRONMENT INFO
Ubuntu 18
Cuda 10.0
tensorflow-gpu=1.15.5

![image](https://user-images.githubusercontent.com/31275645/127487218-da44faa9-f11e-4e15-91c6-7b17f7fc4b09.png)

**Comment by Mahmood-Hussain** [comment]
RESOLVED!
If someone is wondering I needed to set num_classes=21 (one for background). Then I retrained the model again and it worked like a charm
